store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -8189,6 +8221,54 @@ public void modifyContact(OperationContext octxt, int contactId, ParsedContact p
         }
         return newAddrs;
     }
+    
+    /**
+     * Returns a list of contacts don't exist in any contacts folders.
```

Comment:
not "any contacts folder" but "Emailed Contacts folder"

Reply:
Done